activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -79,22 +79,26 @@ def middleware
         end
 
         def clear(options = nil) # :nodoc:
-          local_cache.clear(options) if local_cache
+          return super unless cache = local_cache
```

Comment:
I was trying to keep the first line consistent between all these methods so it's obvious when something is out of place :)
(... I'm ok with either way if the current is not mergable ...)